src/components/WebPageView.js

```diff
@@ -267,6 +267,7 @@ const WebPageView = ({translate, moment, about, user, view, expandAll}) => {
           }
 
           {about.availableChannel &&
+          about.availableChannel.some(grant => grant.availableLanguage) &&
```

Comment:
We might want another variable name instead of `grant`. Probably `channel` or something.